noop/config.ml

```diff
@@ -0,0 +1,7 @@
+open Mirage
+
+let main =
+  foreign "Unikernel.Main" (job)
```

Comment:
Yup, I left them in to make the pattern clearer when progressing to a unikernel with devices. But I agree, it's rubbish style and probably misleading. Will remove.